services/transactionpool/validate_transation.go

```diff
@@ -0,0 +1,13 @@
+package transactionpool
+
+import (
+	"github.com/orbs-network/orbs-spec/types/go/protocol"
+)
+
+func (s *service) validateTransaction(input *protocol.SignedTransaction) (bool, error) {
```

Comment:
Consider combining the bool and the error to one return value - error, because every time we have false we need to give a reason in the error.. so the bool is redundant